clis/teliod/example_teliod_config.json

```diff
@@ -3,5 +3,6 @@
     "log_file_path": "example_log_file.log",
     "authentication_token": "aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa",
     "app_user_uid": "aaaaaaaa-bbbb-cccc-dddd-eeeeeeeeeeee",
-    "interface_name": "utun10"
+    "interface_name": "utun10",
+    "interface_config_provider": "manual"
```

Comment:
love the idea of this, but maybe it makes more sence to use:
```json
"interface": {
   "provider": "manual",
   "name": "utun10",
   ...
}
```
Or is there a requirement to keep this flat ?

Reply:
Good idea, i don't think there is a requirement, but I'll try this out